lib/util.js

```diff
@@ -314,20 +314,10 @@ exports.encode_uri = encode_uri;
  */
 function fetch_rss_entries (feed, max_items=Infinity, max_days_old=1) {
     // Allow environment variables to override this
-    try {
-        if (process.env.LIBINGESTER_MAX_ITEMS) {
-            max_items = parseInt(process.env.LIBINGESTER_MAX_ITEMS);
-        }
-        if (process.env.LIBINGESTER_MAX_DAYS_OLD) {
-            max_days_old = parseInt(process.env.LIBINGESTER_MAX_DAYS_OLD);
-        }
-    } catch (err) {
-        throw new Error(`Couldn't parse RSS entry overrides: ${err}`);
-    }
-
-    const oldest_date = moment().subtract(max_days_old, 'days');
```

Comment:
Old code used the external `moment` library to achieve this. I am happy to change it, but we should also remove the dependency and the require in that case.

Reply:
Good catch, I wanted to remove moment too and forgot about it in the last minute.